ui/apps/dev-dashboard/src/extensions/delete-dev-key.tsx

```diff
@@ -23,27 +23,27 @@ const DeleteDevKeyModal = (props: RootExtensionProps) => {
 
   const navigateTo = plugins['@akashaorg/app-routing']?.routing?.navigateTo;
 
-  const pubKey = React.useMemo(() => {
-    if (extensionData.hasOwnProperty('pubKey') && typeof extensionData.pubKey === 'string') {
-      return extensionData.pubKey;
-    }
-  }, [extensionData]);
-
   const keyName = React.useMemo(() => {
     if (extensionData.hasOwnProperty('keyName') && typeof extensionData.keyName === 'string') {
       return extensionData.keyName;
     }
   }, [extensionData]);
 
-  const deleteKeyMutation = useDeleteDevKey();
+  const deleteKeyMutation = {
```

Comment:
I would prefer to use a `null` value instead of a mock because it's easier to spot afterwards

Reply:
can you revisit every hook and replace that mock with null?
example:
`const myHook = null;`

Reply:
I passed the mock so it takes into consideration other states of the hook like isSuccess, isFetching etc.